journal/src/test/java/io/zeebe/journal/file/JournalTest.java

```diff
@@ -359,6 +360,23 @@ public void shouldOpenAndClose() throws Exception {
     assertThat(journal.isOpen()).isFalse();
   }
 
+  @Test
+  public void shouldReopenJournalWithExistingRecords() throws Exception {
+    // given
+    journal.append(data);
+    journal.append(data);
+    final long lastIndexBeforeClose = journal.getLastIndex();
+    assertThat(lastIndexBeforeClose).isEqualTo(2);
+    journal.close();
+    assertThat(journal.isOpen()).isFalse();
```

Comment:
I'm not sure it's necessary to assert anything here, unless this isn't tested somewhere else?